pre-receive.sh

```diff
@@ -202,7 +202,7 @@ function EXIT_SEDATED() {
 # Check if a line contains a filename from patch file
 # $1 is the line to be checked
 function CHECK_IF_LINE_CONTAINS_FILENAME() {
-  temp_filename=$(echo "$1" | grep -Po '^\+\+\+ b/\K(.*)')
+  local temp_filename=$(echo "$1" | grep -Po '^\+\+\+ b/\K(.*)')
```

Comment:
@wargio could you remove the local variable assignment from this line (205)? This variable is intentionally global.

Reply:
but that variable `temp_filename` is used only there!

Reply:
Checkout line 422 in that file. It is used in the `if` statement.
`if [[ ! "$temp_filename" ]]; then`

Reply:
looks like my fork was older. i'll remove the local.